selftests/vmimage/variants/vmimage.py

```diff
@@ -1,13 +1,26 @@
 import os
 import shutil
+import unittest
 from urllib.error import HTTPError
 from urllib.request import urlopen
 
 from avocado import Test, fail_on
 from avocado.plugins import vmimage as vmimage_plug
-from avocado.utils import process, vmimage
+from avocado.utils import path, process, vmimage
 
 
+def missing_binary(binary):
+    try:
+        path.find_command(binary)
+        return False
+    except path.CmdNotFoundError:
+        return True
+
+
+@unittest.skipIf(
+    missing_binary("qemu-img"),
```

Comment:
IIUIC these tests are not dependent on `qemu-img` binary, therefore we should remove this skip.